SU2_CFD/src/solvers/CIncEulerSolver.cpp

```diff
@@ -1081,6 +1101,8 @@ void CIncEulerSolver::Upwind_Residual(CGeometry *geometry, CSolver **solver_cont
   const bool muscl      = (config->GetMUSCL_Flow() && (iMesh == MESH_0));
   const bool limiter    = (config->GetKind_SlopeLimit_Flow() != NO_LIMITER);
   const bool van_albada = (config->GetKind_SlopeLimit_Flow() == VAN_ALBADA_EDGE);
+  const bool energy     = config->GetEnergy_Equation();
+  const bool flame      = (config->GetKind_Scalar_Model() == PROGRESS_VARIABLE);
```

Comment:
@danielmayer Please write a few words (in the PR) describing this interaction between the flamelet solver and the energy equation in incompressible flows.
Replaces? Overrides? Requires it?
If possible encapsulate the `energy && !flame` logic in GetEnergy_Equation, that would be the safest way to guarantee that new pull requests will not break, and will be compatible, with this feature.

Reply:
we now keep the temperature field but do not solve anything for temperature. We look up temperature and then store it in the temperature field.